bin/docbookrx

```diff
@@ -1,13 +1,13 @@
 #!/usr/bin/env ruby
 
-if File.exist?(app_lib_path = File.join(File.dirname(__FILE__), '../lib/docbookrx'))
+
+if File.exist?(app_lib_path = File.expand_path(File.join(File.dirname(__FILE__), '../lib/docbookrx')))
```

Comment:
Like I said in the commit message, I got an exception when trying to run the script (not installed), directly from the root source directory:

/usr/share/rubygems/rubygems/core_ext/kernel_require.rb:54:in `require':
     cannot load such file -- bin/../lib/docbookrx (LoadError)
        from /usr/share/rubygems/rubygems/core_ext/kernel_require.rb:54:in `require'
        from bin/docbookrx:4:in `<main>'

By forcing the path to be absolute (no /../), it works fine.